.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,4 @@
+contact_links:
+    - name: Usage question
+      url: https://neurostars.org/
```

It's probably worth it to link directly to the tag's page. When you create a new topic from the tag's page, it automatically adds the tag to the post.

```suggestion
      url: https://neurostars.org/tag/nilearn
```